ai-assistant-product/README_DE.md

```diff
@@ -1 +1,667 @@
 # ai-assistant
+
+<todo>
+
+## Demo
+
+### Demoprojekt
+
+Im **ai-assistant-demo** Ivy-Projekt haben wir Demo-Assistenten erstellt, die Ihnen helfen können, Ihren eigenen KI-Assistenten effizienter zu verstehen und zu entwickeln.
+
+> [!IMPORTANT]
+> Dieses Demoprojekt wird Ivy-Benutzer und -Rollen erstellen und die Hauptvariablen des KI-Assistenten überschreiben. Es wird daher DRINGEND EMPFOHLEN, es auf einer Ivy-Engine im Demomodus auszuführen, um Ihre Daten zu schützen.
+>
+> Wenn Sie dieses Demo in einer Produktionsumgebung ausführen möchten, stellen Sie bitte sicher, dass Sie alle Ivy-Variablen, die mit dem KI-Assistenten in Verbindung stehen, sichern und nach Abschluss des Demos wiederherstellen.
+
+#### Komplexe Demo
+
+**Anwendungsfall**
+
+Der Benutzer kann ein Softwareprojekt mit dem KI-Assistenten planen. Der KI-Assistent ermöglicht es dem Benutzer, ein Softwareprojekt effizient zu planen, indem er Informationen in ein strukturiertes Format organisiert. Er kann:
+
+- Das Projekt erstellen und in der Datenbank speichern
+- Bei der Suche und Auswahl geeigneter Teammitglieder für das Projekt unterstützen
+- Dem Benutzer bei der Vorbereitung des Kick-off-Meetings für das Projekt helfen
+- Aufgaben erstellen und zuweisen, um Teammitglieder zu dem Meeting einzuladen
+
+**Wie man es verwendet**
+
+1. Führen Sie den Prozess `startComplexDemo` aus, um Testdaten zu erstellen und den ursprünglichen KI-Assistenten durch den **Complex Demo Assistant** zu ersetzen
+
+2. Öffnen Sie im **Portal** die KI-Assistenten-App
+
+3. Jetzt können Sie den Assistenten verwenden, um ein Softwareprojekt zu planen und zu erstellen, indem Sie die Details des Projekts eingeben, z. B. die Anzahl der Mitglieder oder die Technologien.
+
+Beispiel:
+
+`The XYZ Solutions Web Development Project aims to build a state-of-the-art web application that enhances user experience and supports the company's growth objectives. Mainly use basic web technologies such as HTML and CSS, the project will deliver a secure, scalable, and efficient platform that aligns with industry standards and best practices. We need 3 engineer, 1 web designer and 2 tester for it.`
```

Comment:
Isn't it possible to ask a question in German?

Reply:
Exactly.
Currently it's unstable if asking in German, we will implement better multi language approach when using AI Flow in near future.